### DISCUSSION TITLE
Wither attacking animals / Wither rose farm

### DISCUSSION THREAD

**Discussion by x1p** [GriefPrevention/GriefPrevention#738] [Ideas]
**As** a player it nice to build a wither rose farm
**I want** that the wither can attack animals if desired which is impossible in land claims because animals are invulnerable.
**Because** A wither should attack chickens to get the rose

There is a design to make a wither rose farm. It's made of a isolation cell for the wither were it cannot escape. Chickens are spawned in this area using eggs from a dispenser. The chickens are attacked. The witherrose drops on carpet. Hoppers beneath it should pick it up.
![Screenshot_20200301_221539](https://user-images.githubusercontent.com/28690790/75634049-70caf900-5c0a-11ea-8c07-e72b3af8a0b2.png)

This farm works best with chickens. There easy to produce and the wither attacks them unlike mobs.
The problem is that this only works in unclaimed land otherwise chickens are immune.
If this is made in unclaimed land, it is vulnerable to grieving.

Optional questions:

**Is your feature request related to a problem? Please describe.**
Naturally acquiring wither roses in a farm like design within a land claim

**Describe alternatives you've considered**
Make an unclaimed area although it is vulnerable for grieving.

I realize this is not easy to solve unless there is a switch to toggle for players if withers should attacks animals or not.

**Comment by zeddash** [comment]
Just had to modify my massive claim into 4 claim with a hole in the middle for the killing chamber, would have appreciated being able to enable mob killing in a subdivision

**Comment by brasilcraft54** [comment]
am i really having a problem with this? i really wish my players could do rose farm wither
